docs/ABOUT.md

```diff
@@ -0,0 +1,46 @@
+[Dart](https://en.wikipedia.org/wiki/Dart_(programming_language)) is a general-purpose computer programming language that is:
+ 1. [Concurrent](https://en.wikipedia.org/wiki/Concurrent_computing)
+ 2. [Class-Based](https://en.wikipedia.org/wiki/Class-based_programming)
+ 3. [Object-Oriented](https://en.wikipedia.org/wiki/Object-oriented_programming)
+
+Dart applications are typically compiled to [bytecode](https://en.wikipedia.org/wiki/Bytecode) that can run on any [Dart virtual machine](https://en.wikipedia.org/wiki/Dart_virtual_machine) (JVM) regardless of computer architecture.
+
+The primary concepts baked into the making of the language were:
+
+ - It is simple, object-oriented, and familiar.
+ - It provides tools out of the box to help development efforts.
+ - It is architecture-neutral and portable.
+ - It executes with high performance for practical applications and not just benchmarks.
+
+The language derives its syntax from Java and JavaScript; as well as Smalltalk.
```

Comment:
I wonder if this would be more compelling if it were part of the bit about the language being familiar.